web/css/json-editor.css

```diff
@@ -111,7 +111,10 @@ a:hover, a:focus {
 .sg-snippet-resize-handle.pos-right {
     border-right: 1px solid #ccc;
     left: -40px;
-    cursor: e-resize
+    cursor: e-resize;
+    background: #f8f4fe;
+    height: 98%;
```

Comment:
I was just playing with this in-browser, the `top: 1%` is a bit inconsistent in my viewport.  What would you think of:

```
height: 100%;
top: 0;
margin: 0;
```

And adding this to `.sg-snippet-resize-handle.pos-right:before`:

```
top: 50vh;
```

^ This would keep the icon in the viewport no matter how tall the bar gets.